common/djangoapps/course_modes/views.py

```diff
@@ -148,6 +150,21 @@ def get(self, request, course_id, error=None):
             "responsive": True,
             "nav_hidden": True,
         }
+
+        if enterprise_enabled():
+            enterprise_learner_data = EnterpriseApiClient().fetch_enterprise_learner_data(user=request.user)['results']
```

Comment:
`EnterpriseApiClient()` itself contains the method -- `fetch_enterprise_learner_data`. What will be the purpose of introducing a new helper method that further calls the `EnterpriseApiClient().fetch_enterprise_learner_data` ?